packages/backend/src/routers/apiV1Router.ts

```diff
@@ -146,6 +146,21 @@ apiV1Router.get(
         scope: ['profile', 'email'],
     }),
 );
+apiV1Router.get(
+    '/login/gdrive',
+    storeOIDCRedirect,
+    passport.authenticate('google', {
+        scope: [
+            'profile',
+            'email',
+            'https://www.googleapis.com/auth/drive.metadata.readonly',
+        ],
+        accessType: 'offline',
+        prompt: 'consent',
+        session: false,
+        includeGrantedScopes: true,
```

Comment:
more importantly, should we also have `includeGrantedScopes` on the original auth endpoint? 

In the case that the user authorizes drive scope and then later logs in again on the login endpoint (which doesn't include the drive scope). We don't want to lose it.

Reply:
Let's add this `includeGrantedScopes: true,` to the other google endpoint